website/docs/r/data_lake_store.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_lake_store"
+sidebar_current: "docs-azurerm-resource-data-lake-store"
+description: |-
+  Create a Azure Data Lake Store.
+---
+
+# azurerm\_data\_lake\_store
+
+Create a Azure Data Lake Store.
+
+## Example Usage
+
+```hcl
+# Pay As You Go
+resource "azurerm_resource_group" "test" {
+    name = "test"
+    location = "northeurope"
+}
+
+resource "azurerm_data_lake_store" "payasyougo" {
```

Comment:
Can we change this to consumption rather the payasyougo to better match the tier name in azure?